src/components/countdown/code/lib/countdown.js

```diff
@@ -0,0 +1,56 @@
+var moment = require('moment');
+var Countdown = {};
+module.exports = Countdown;
+
+var pad = function (number) {
+  number = Math.abs(Math.floor(number));
+  if (number > 9) {
+    return number;
+  } else {
+    return '0' + number;
+  }
+};
+
+var roundTowardsZero = function(number) {
```

Comment:
These functions should be prepended with an underscore and exported so they can be tested independently.